src/main/resources/db/migration/V1_21__associate_referral_with_intervention.sql

```diff
@@ -0,0 +1,3 @@
+alter table referral
+    add column intervention_id uuid,
+    add constraint fk_intervention_id foreign key (intervention_id) references intervention;
```

Comment:
Should this have a not null?

Reply:
now we have dev data, this is done.

Reply:
I don't understand the default value on the intervention_id - wouldn't this fail anyway as the intervention with that id ('0000...') doesn't exist?

Reply:
Has the not null been removed accidentally in this last update?